arrow/Cargo.toml

```
@@ -38,10 +38,10 @@ path = "src/lib.rs"
 bench = false
 
 [target.'cfg(target_arch = "wasm32")'.dependencies]
-ahash = { version = "0.8", default-features = false, features = ["compile-time-rng"] }
+ahash = { version = "0.8.0", default-features = false, features = ["compile-time-rng"] }
```

```suggestion
ahash = { version = "=0.8.0", default-features = false, features = ["compile-time-rng"] }
```

FYI you will need an exact version pin, cargo applies semver compatibility by default